javascript/builtins/Map.json

```
@@ -473,6 +473,74 @@
             }
           }
         },
+        "groupBy": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Map/groupBy",
+            "spec_url": "https://tc39.es/proposal-array-grouping/#sec-map.groupby",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": [
+                {
+                  "version_added": "preview",
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "javascript.options.experimental.array_grouping",
+                      "value_to_set": "true"
+                    }
+                  ]
+                },
+                {
+                  "version_added": "98",
+                  "version_removed": "preview",
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "javascript.options.experimental.array_grouping",
+                      "value_to_set": "true"
+                    }
+                  ],
+                  "alternative_name": "Array.prototype.groupToMap"
+                }
+              ],
```

The gods of BCD might take a different view, but I see no value in keeping info about the removed preferenced API in FF

```suggestion
              "firefox": {
                "version_added": "preview",
                "flags": [
                  {
                    "type": "preference",
                    "name": "javascript.options.experimental.array_grouping",
                    "value_to_set": "true"
                  }
                ]
              },
```

---

FYI @queengooborg @Elchi3 or whoever reviews this, the preference info here is correct - it's a preference that only exists in whatever the nightly build is, and which is false by default.